### PR TITLE
feat(Brand): consumed Penta updates

### LINKED ISSUE #9921: Consume core Penta updates: Brand

### PR DESCRIPTION
**Pull request by mfrances17**
**What**: Closes #9921 

Mostly icon changes, added some extra code to make it responsive in different screen sizes and to make it work with light/dark mode.

**Additional issues**:
None

### CODE REVIEW

**Review by andrew-ronaldson — APPROVED** [review]
Approved. Great to have the light/dark switcher working on this too!

File: packages/react-core/src/components/Brand/examples/BrandBasic.tsx [hunk 1]
```diff
@@ -1,7 +1,15 @@
...
+import pfLogoDark from '../../assets/PF-HorizontalLogo-Reverse.svg';
 
 export const BrandBasic: React.FunctionComponent = () => (
-  <Brand src={pfLogo} alt="Patternfly Logo" widths={{ default: '200px', md: '400px', xl: '600px' }} />
+  <>
+    <div className="show-light">
+      <Brand src={pfLogo} alt="Patternfly Logo" widths={{ default: '200px', md: '400px', xl: '600px' }} />
+    </div>
+    <div className="show-dark">
+      <Brand src={pfLogoDark} alt="Patternfly Logo" widths={{ default: '200px', md: '400px', xl: '600px' }} />
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Brand/examples/BrandBasic.tsx:12]: For the alt text in these examples, can we just omit the word "logos", so just `alt="PatternFly"`.

File: packages/react-core/src/components/Brand/examples/BrandResponsive.tsx [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
-  <Brand src={logoBase} alt="Fallback patternfly default logo" widths={{ default: '40px', sm: '60px', md: '220px' }}>
-    <source media="(min-width: 1200px)" srcSet={logoXl} />
-    <source media="(min-width: 992px)" srcSet={logoLg} />
-    <source media="(min-width: 768px)" srcSet={logoMd} />
-    <source media="(min-width: 576px)" srcSet={logoSm} />
-    <source srcSet={logo} />
-  </Brand>
+  <>
+    <div className="show-light">
+      <Brand src={pfLogo} alt="Fallback patternfly default logo" widths={{ default: '40px', sm: '60px', md: '220px' }}>
```
Comment (thatblindgeye) [inline packages/react-core/src/components/Brand/examples/BrandResponsive.tsx:12]: Similarly we should just pass "PatternFly" as the alt text here.

**Review by thatblindgeye — COMMENTED** [review]
Couple of quick comments below. @edonehoo do you think it'd be worth adding a description to the basic example to call out that theme support has to be manually done by the consumer, rather than it being something built into the component?

File: packages/react-core/src/components/Brand/examples/BrandBasic.tsx (hunk 1 above)
Comment (mfrances17) [reply packages/react-core/src/components/Brand/examples/BrandBasic.tsx]: sounds good, done!

**Review by mcoker — APPROVED** [review]
LGTM 💪

File: packages/react-core/src/components/Brand/examples/BrandResponsive.tsx (hunk 2 above)
Comment (thatblindgeye) [reply packages/react-core/src/components/Brand/examples/BrandResponsive.tsx:12]: The `show-dark` one below looks good, we just need this one's alt text updated as well

**Comment by edonehoo** [comment]
> Couple of quick comments below. @edonehoo do you think it'd be worth adding a description to the basic example to call out that theme support has to be manually done by the consumer, rather than it being something built into the component?

@thatblindgeye I think that makes sense here! wdyt about something simple like this?:

> The brand component does not have any built-in theme support. If a brand should be dynamically styled, you must manually set up the necessary logic on your own. 
> 
> This example adjusts styling as the PatternFly website is toggled between the light and dark theme.

**Review by wise-king-sullyman — APPROVED** [review]
Other than the above comments LGTM!

**Comment by mfrances17** [comment]
> @thatblindgeye I think that makes sense here! wdyt about something simple like this?:
> 
> > The brand component does not have any built-in theme support. If a brand should be dynamically styled, you must manually set up the necessary logic on your own.
> > This example adjusts styling as the PatternFly website is toggled between the light and dark theme.

I added this, slight wording changes and placement... I added the blurb above the basic example under Examples because it applies to both examples.

File: packages/react-core/src/components/Brand/examples/BrandResponsive.tsx (hunk 2 above)
Comment (mfrances17) [reply packages/react-core/src/components/Brand/examples/BrandResponsive.tsx]: Strange, I must've done too many undos or something cuz I know it was done at some point... 😄 thanks for catching! Done!

**Review by thatblindgeye — APPROVED** [review]
🎉

**Review by tlabaj — APPROVED** [review]
LGTM!

**Pull request merged**